pkg/util/i18n/translations/en-us.go

```diff
@@ -0,0 +1,16 @@
+package translations
+
+var EnUSFileName = "en-US.all.json"
+
+var EnUSData = []byte(`
```

Comment:
Yes. Using a better-established format for the translation files favours translators that are not programmers and a world of tools that exist to support them. JSON and YAML are usually acceptable but something closer to plain text is usually better, like Java or Mozilla .properties, CSVs like `id,translated text` and so on. Conversions scripts would probably also work, though.